StatusPlayer.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import boto3
+import json
+from boto3.dynamodb.conditions import Attr
+
+# https://support.twilio.com/hc/en-us/articles/223181468-How-do-I-Add-a-Line-Break-in-my-SMS-or-MMS-Message-
+CRLB = "%0a"
+
+def handler(event, context):
+    logging.info('Handling event {} - context {}', event, context)
+
+    dynamodb = boto3.resource('dynamodb', region_name='us-east-1')
+    player_table = dynamodb.Table('HangmanPlayer')
+    game_table = dynamodb.Table('HangmanGame')
+
+    response = game_table.scan(
+        FilterExpression=Attr('gameState').ne('over')
+    )
+    running_games = list()
+    for game in response['Items']:
+        if game['gameState'] == 'running':
+            running_games.append(game)
+
+    game = running_games[0]
+    phone_number = event['data']['twilio']['From']
+    response = player_table.get_item(
+        Key={
+            'phoneNumber': phone_number,
+            'gameId': game['Id']
+        }
+    )
+
+    # Get the number of lives remaining
+    player_info = response['Item']
```

Comment:
Don't we need to get the zeroth'd indexed item here if it exists? It's possible the game+player aren't present.

Reply:
Added  a check for the existence of a running game and a check that the player is registered to that game.